feat(react-scheduler): add capability to render zero time appointments (#2406)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/docs/guides/appointments.md

.embedded-demo({ "path": "scheduler-basic/day", "showThemeSelector": true })

## Zero-Time Appointments

> Zero-time appointments are appointments with specified only start date and time. They show instantaneous actions such as reminders, public transportation scheduler and so on. To display zero-time appointments on a view you should set only the `startDate` field in [AppointmentModel](../reference/scheduler.md#appointmentmodel) shape and add the [Appointments](../reference/appointments.md) plugin.

Reviewer: ```suggestion
Zero-time appointments do not have an end date and time. They represent events without duration (reminders, arrival/departure time, and so on).

To add a zero-time appointment, set the `startDate` field in the [AppointmentModel](../reference/scheduler.md#appointmentmodel), leave the `endDate` field unspecified, and add the [Appointments](../reference/appointments.md) plugin.
```

.embedded-demo({ "path": "scheduler-basic/day", "showThemeSelector": true })

## Zero-Time Appointments

> Zero-time appointments do not have an end date and time. They represent events without duration (reminders, arrival/departure time, and so on).

Reviewer 2: ```suggestion
Zero-time appointments do not have an end time and date. They represent events without duration (reminders, arrival/departure time, and so on).
```

> To add a zero-time appointment, set the `startDate` field in the [AppointmentModel](../reference/scheduler.md#appointmentmodel), leave the `endDate` field unspecified, and add the [Appointments](../reference/appointments.md) plugin.

Reviewer 2: ```suggestion
To add a zero-time appointment, specify the `startDate` in the [AppointmentModel](../reference/scheduler.md#appointmentmodel), leave the `endDate` field unspecified, and add the [Appointments](../reference/appointments.md) plugin.
```

.embedded-demo({ "path": "scheduler-basic/zero-time", "showThemeSelector": true })

## All-Day Appointments

> All-day appointments last for the entire day or multiple days. They are displayed only on the [MonthView](../reference/month-view.md). To display them on the day or week views, use the [AllDayPanel](../reference/all-day-panel.md) plugin.

Reviewer 2: ```suggestion
All-day appointments last for the entire day or multiple days. They are displayed only in the [MonthView](../reference/month-view.md). To display them in the day or week views, use the [AllDayPanel](../reference/all-day-panel.md) plugin.
```
